README.md

```diff
@@ -56,3 +57,26 @@ Check the validity of the ecosystem yaml file
         additional_dependencies:
           - cerberus
 ```
+
+### remove-improper-eol-in-cn-docs
+
+Remove end-of-line characters that split natural paragraphs in Chinese docs.
+
+This helps resolve extra whitespaces in Chinese Markdown docs described [here](https://stackoverflow.com/questions/8550112/prevent-workaround-browser-converting-n-between-lines-into-space-for-chinese/8551033#8551033), as a long-standing HTML rendering issue. For example,
+
+> 这是一个，
+> 像诗一样的
+> 测试
+
+will be changed to:
+
+> 这是一个，像诗一样的测试
+
+Usage:
+
+```yaml
+  - repo: https://github.com/open-mmlab/pre-commit-hooks
+    rev: v0.3.0
```

Comment:
```suggestion
    rev: v0.4.0
```

Reply:
After merging the PR, we will bump the version to v0.4.0.

Reply:
The version v.3.0 in [here](https://github.com/open-mmlab/pre-commit-hooks/blob/master/README.md#using-pre-commit-hooks-with-pre-commit) also needs to be upgraded to v0.4.0.